src/edge.ts

```diff
@@ -28,39 +29,57 @@ const genId = () => {
     .join('');
 };
 
-function getInstance(params?: ConfigParameters): Auth0Edge {
+const telemetry = { name: 'nextjs-auth0', version };
```

Comment:
Should this be moved to some common helper, as it's also used in https://github.com/auth0/nextjs-auth0/blob/141ec7bff57684747e99baddf02840706adda33b/src/index.ts#L26?

Reply:
Yeah, I can move this to `shared.ts`